crates/http/src/routes.rs

```diff
@@ -147,6 +148,15 @@ impl RoutePattern {
     }
 }
 
+impl fmt::Display for RoutePattern {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match &self {
+            RoutePattern::Exact(path) => write!(f, "{}", path),
+            RoutePattern::Wildcard(pattern) => write!(f, "{}", pattern),
```

Comment:
I think there should be a way to indicate this is a wildcard pattern — how about `/...`? We use it already to define such a pattern anyway.

Reply:
Yeah, I agree to distinguish wildcard pattern from exact path, but using `/...` identifier will cause it unclickable from the terminal. Is that fine?

Reply:
Ah, good point.
Perhaps something like: `write!(f, "{} (wildcard)", pattern)` ?

(I don't know what the best solution would be here either.)

Reply:
Oh, that's the exact same idea when I first implemented this functionality :)
I take this solution here and hope someone will come up with the best solution. Done.